src/legacy/core_plugins/kibana/public/visualize/embeddable/disabled_lab_visualization.tsx

```diff
@@ -27,10 +28,17 @@ export function DisabledLabVisualization({ title }: { title: string }) {
         aria-hidden="true"
       />
       <div className="kuiVerticalRhythm">
-        <em className="visDisabledLabVisualization__title">{title}</em> is a lab visualization.
```

Comment:
Not sure if you care about these class names or the `<em>`, but just realized they are no longer there with the `<FormattedMessage>`

Reply:
Didn't care enough to try to keep them in. :)  @azasypkin - in your i18n experiences has this situation been dealt with?  I figured there was not an easy way to add inline styling but I could be wrong!

Reply:
Yeah, the current state of things isn't ideal @Stacey-Gammon. But in react you can use JSX in values, so in this case it would look like this:
```tsx
<FormattedMessage
  id="kbn.visualize.disabledLabVisualizationTitle"
  defaultMessage="{title} is a lab visualization."
  values={{ title: <em className="visDisabledLabVisualization__title">{title}</em> }}
/>
```

Reply:
cool, thanks!